### ISSUE TITLE
Message exceeds maximum length and will be dropped

### ISSUE THREAD

**Issue by qhh0205** [logstash-plugins/logstash-output-sqs#23]
- Logstash Version: logstash 5.2.1
- logstash-output-sqs version: logstash-output-sqs 4.0.0
- Operating System: CentOS7
**I observed the logstash log, there are the following warning, will this lead to data loss?**
`[2017-02-22T16:33:32,487][WARN ][logstash.outputs.sqs     ] Message exceeds maximum length and will be dropped {:message_size=>462008}
[2017-02-22T16:36:13,334][WARN ][logstash.outputs.sqs     ] Message exceeds maximum length and will be dropped {:message_size=>465169}`

**Comment by joshuaspence** [comment]
Yes this leads to data loss. SQS cannot handle messages larger than 256KB.

**Comment by qhh0205** [comment]
But i only use the SQS to sync the log，in fact, the message size exceeds  256KB is impossible.
